packages/core/content-type-builder/admin/src/components/ComponentCard/index.js

```diff
@@ -97,7 +103,7 @@ function ComponentCard({ component, dzName, index, isActive, isInDevelopmentMode
       </Box>
 
       {isInDevelopmentMode && (
-        <CloseButton as="button" onClick={onClose} type="button">
+        <CloseButton as="span" onClick={onClose} type="button">
```

Comment:
Could you explain this change? it seems clickable, so why should it not be a button anymore?

Reply:
Is it one of those situations where you click a card and the button is just there to look like a button if they don't click the card? If so, I think its fine - if its not and it does something different we should review this design // implementation.

Reply:
thanks for explaining, then assigning to a span seems good to me, but I believe you should remove the `type="button"`

Reply:
Sounds ok to me, but I think you'd have to make sure the `div` behaves like a button then (e.g. adding tabindex, to make it focusable).